StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/NoXmlFileHeaderUnitTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.DocumentationRules;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// Unit tests for file header that do not follow the XML syntax.
+    /// </summary>
+    public class NoXmlFileHeaderUnitTests : DiagnosticVerifier
+    {
+        private const string SettingsFileName = "stylecop.json";
+        private const string TestSettings = @"
+{
+    ""settings"": {
+    ""documentationRules"": {
+      ""copyrightText"": ""Copyright (c) FooCorp. All rights reserved.\nLicensed under the ??? license. See LICENSE file in the project root for full license information."",
+      ""xmlHeader"": false
+    }
+  }
+}
+";
+
+        /// <summary>
+        /// Verifies that a file header with an autogenerated comment will not produce a diagnostic message.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task TestAutoGeneratedSourceFileAsync()
```

Comment:
:arrow_right: We can remove this test later if we decide it's truly a duplicate.